VideoWeb/VideoWeb/ClientApp/src/app/security/security-config-setup.service.ts

```diff
@@ -69,13 +69,12 @@ export class SecurityConfigSetupService {
             this._configSetupSubject.pipe(filter(Boolean), first()).subscribe(() => {
                 this.oidcConfigService.withConfig(this.config[provider]);
             });
-            this.currentIdpSubject.next(provider);
-            this._configRestoredSubject.next(true);
         }
+        this.currentIdpSubject.next(provider);
+        this._configRestoredSubject.next(true);
```

Comment:
This should have been outside the if statement. The issue was it was not set during setup for quick link users